res/themes/light/css/_light.scss

```diff
@@ -198,9 +198,8 @@ $event-highlight-bg-color: $yellow-background;
 $event-timestamp-color: #acacac;
 
 $copy-button-url: "$(res)/img/feather-customised/clipboard.svg";
-$collapse-button-url: "$(res)/img/feather-customised/minimise.svg";
-$expand-button-url: "$(res)/img/feather-customised/maximise.svg";
-
```

Comment:
Let's keep this blank line separator, it's useful to break up the sections a bit.